src/modules/GroupChannelList/components/GroupChannelListUI/GroupChannelListUIView.tsx

```diff
@@ -27,6 +27,9 @@ export interface Props {
   renderChannel: (props: { item: GroupChannel; index: number }) => React.ReactElement;
 
   renderAddChannel(): React.ReactElement;
+
+  // NOTE: We only support scrollRef in GroupChannelList
```

Comment:
What does it mean by this? Could you elaborate more?

Reply:
oh, yes, let me make it clearly